app/models/user.rb

```diff
@@ -8,7 +9,7 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable, authentication_keys: [:username]
 
-  delegate :superadmin, :admin, :game_master, :player, to: :user_roles
+  delegate :superadmin, :admin, :game_master, :player, to: :user_roles, prefix: true
```

Comment:
Added prefix becuase now we have model wich name is `game_master` too. So without prefix delegated method `game_master` will override model `GameMaster` method